src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -65,6 +66,22 @@ import { EnsObject } from 'src/shared/state/ens-object/ensObjectTypes';
 
 import styles from './Browser.scss';
 
+const client = new ApolloClient({
+  uri: '/thoas',
+  cache: new InMemoryCache({
+    typePolicies: {
+      Gene: {
+        keyFields: ['stable_id'],
+        fields: {
+          slice: {
+            merge: false
+          }
+        }
+      }
+    }
+  })
+});
```

Comment:
After Imran's PR got accepted, this should no longer be necessary. See [EntityViewer component](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx).

I am also wondering whether it might not be a good idea to separate, inside the Browser component, the Browser from, say, ApolloWrappedBrowser, as two different exports. The default export would be Apollo-wrapped, but the export for unit-testing won't have any notion of Apollo. Of course that would mean mocking its children components that will complain of the absence of the Apollo context.

Kinda what our App.tsx [used to do](https://github.com/Ensembl/ensembl-client/blob/c3f5a8520ccbd255364ee894a687491dafa75d7b/src/ensembl/src/content/app/App.tsx#L92-L111) a long time ago.

Reply:
I have made the changes to import the client and removed the lines above. Is it necessary to export Browser without any notation of Apollo right now?

Reply:
What's the purpose of still mocking apollo-client in the test file?

![image](https://user-images.githubusercontent.com/6834224/107514699-ea50c280-6ba1-11eb-8124-a3b872bd9f99.png)

Reply:
I get the following error when I don't:
<img width="1079" alt="Screenshot 2021-02-10 at 13 48 47" src="https://user-images.githubusercontent.com/3085815/107518385-d78cbc80-6ba6-11eb-872a-85f5de16f2bc.png">

But you are right that we shouldn't be mocking it here.

Reply:
That is so strange! Mocking of `'src/gql-client'` should have gotten rid of this problem. But I am seeing the same error locally as well...

Reply:
this works for me: `jest.mock('src/gql-client');` -> `jest.mock('src/gql-client', () => ({ client: jest.fn() }));`.

Don't understand why jest isn't doing this automatically, but there we are.